tests/test_models/test_deploy_wrappers.py

```diff
@@ -0,0 +1,221 @@
+                in_channels=256,
+                loss=dict(type='DBLoss', alpha=5.0, beta=10.0,
+                          bbce_loss=True)),
+            train_cfg=None,
+            test_cfg=None))
+
+    cfg = mmcv.Config(cfg)
+
+    pytorch_model = build_detector(cfg.model, None, None)
+
+    # prepare data
+    inputs = torch.rand(1, 3, 224, 224)
+    img_metas = [{
+        'img_shape': [1, 3, 224, 224],
+        'ori_shape': [1, 3, 224, 224],
+        'pad_shape': [1, 3, 224, 224],
+        'filename': None,
+        'scale_factor': np.array([1, 1, 1, 1])
+    }]
+
+    pytorch_model.forward = pytorch_model.forward_dummy
+    with torch.no_grad():
+        torch.onnx.export(
+            pytorch_model,
+            inputs,
+            onnx_path,
+            input_names=['input'],
+            output_names=['output'],
+            export_params=True,
+            keep_initializers_as_inputs=False,
+            verbose=False,
+            opset_version=11)
+
+    # TensorRT part
+    def get_GiB(x: int):
+        """return x GiB."""
+        return x * (1 << 30)
+
+    trt_path = onnx_path.replace('.onnx', '.trt')
+    min_shape = [1, 3, 224, 224]
+    max_shape = [1, 3, 224, 224]
+    # create trt engine and wraper
+    opt_shape_dict = {'input': [min_shape, min_shape, max_shape]}
+    max_workspace_size = get_GiB(1)
+    trt_engine = onnx2trt(
+        onnx_path,
+        opt_shape_dict,
+        log_level=trt.Logger.ERROR,
+        fp16_mode=False,
+        max_workspace_size=max_workspace_size)
+    save_trt_engine(trt_engine, trt_path)
+    print(f'Successfully created TensorRT engine: {trt_path}')
+
+    wrap_onnx = ONNXRuntimeDetector(onnx_path, cfg, 0)
+    wrap_trt = TensorRTDetector(trt_path, cfg, 0)
+    # os.remove(onnx_path)
+    assert isinstance(wrap_onnx, ONNXRuntimeDetector)
+    assert isinstance(wrap_trt, TensorRTDetector)
+
+    with torch.no_grad():
+        onnx_outputs = wrap_onnx.simple_test(inputs, img_metas, rescale=False)
+        trt_outputs = wrap_onnx.simple_test(inputs, img_metas, rescale=False)
+
+    assert isinstance(onnx_outputs[0], dict)
+    assert isinstance(trt_outputs[0], dict)
+    assert 'boundary_result' in onnx_outputs[0]
+    assert 'boundary_result' in trt_outputs[0]
+
+
+@pytest.mark.skipif(torch.__version__ == 'parrots', reason='skip parrots.')
+@pytest.mark.skipif(
+    version.parse(torch.__version__) < version.parse('1.4.0'),
+    reason='skip if torch=1.3.x')
+@pytest.mark.skipif(
+    not torch.cuda.is_available(), reason='skip if on cpu device')
+def test_recognizer_wraper():
+    try:
+        import onnxruntime as ort  # noqa: F401
+        import tensorrt as trt
+        from mmcv.tensorrt import (onnx2trt, save_trt_engine)
+        from mmocr.models.deploy_helper import (ONNXRuntimeRecognizer,
+                                                TensorRTRecognizer)
+    except ImportError:
+        pytest.skip('ONNXRuntime or TensorRT is not available.')
```

Comment:
There is a https://docs.pytest.org/en/latest/reference/reference.html#pytest-importorskip

Reply:
remember to try this

Reply:
Seems it can not pass the following line:
```
onnx2trt = pytest.importorskip('mmcv.tensorrt.onnx2trt')
```
Perhaps importorskip function can only import installed packages.

Reply:
try `@importorskip(...)`

Reply:
It seems to me that importorskip itself works as try import sth

Reply:
Meanwhile, importorskip can only import installed package.

Reply:
will it be shorter?

Reply:
I will take a try for @importorskip(...)

Reply:
Hi, @innerlee. Found suddenly that there is no @pytest.mark.importorskip function but it gives me no error or warning message. Again, checked @pytest.importorskip function, it can not import modules like mmcv.tensorrt or mmcv.tensorrt.tensorrt_utils.onnx2trt. So, we may move back to use it inside unit tests or use try except.

Reply:
This is skipped because there is no cuda.

How about @importorskip?

Reply:
BTW you should run pytest locally in a pc that has onnx, cuda, etc. Github will skip them

Reply:
I actually tested locally.

Reply:
> How about @importorskip?

Well, I importorskip should be a part of pytest.... Can not use @importorskip directly

Reply:
Okay interesting. Please give a fix. Use the original style is okay.

Reply:
Opened a issue [here](https://github.com/pytest-dev/pytest/issues/8776). A fix will be requested soon.